client/my-sites/site-settings/settings-newsletter/main.tsx

```diff
@@ -0,0 +1,114 @@
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import DocumentHead from 'calypso/components/data/document-head';
+import FormattedHeader from 'calypso/components/formatted-header';
+import Main from 'calypso/components/main';
+import isSiteAutomatedTransfer from 'calypso/state/selectors/is-site-automated-transfer';
+import { getSiteUrl, isJetpackSite } from 'calypso/state/sites/selectors';
+import { IAppState } from 'calypso/state/types';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { NewsletterSettingsSection } from '../reading-newsletter-settings';
+import wrapSettingsForm from '../wrap-settings-form';
+
+export type SubscriptionOptions = {
+	invitation: string;
+	comment_follow: string;
+};
+
+type Fields = {
+	subscription_options?: SubscriptionOptions;
+	wpcom_featured_image_in_email?: boolean;
+	wpcom_subscription_emails_use_excerpt?: boolean;
+	sm_enabled?: boolean;
+};
+
+const getFormSettings = ( settings: unknown & Fields ) => {
```

Comment:
Why do we have the union with `unknown` type here? Also, one line below, it seems like you are doing a check for `! settings` which implies to me that the argument is `optional`.